media-libs/libid3tag/libid3tag-0.16.3.recipe

```diff
@@ -34,36 +35,28 @@ BUILD_REQUIRES="
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	cmd:aclocal_1.13
-	cmd:autoconf
+	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
 	cmd:gperf >= 3.1
-	cmd:libtoolize$secondaryArchSuffix
```

Comment:
Nitpicking (otherwise looks good to me), the version string for gperf could be dropped, I guess it's a leftover from having 2 versions for it in the past, only one is in the depot.